src/trajectory_data_handling/sqlite_querying.cpp

```diff
@@ -73,6 +75,13 @@ namespace trajectory_data_handling {
         return 0;
     }
 
+    int query_handler::callback_rtree_datastructure(void *query_success_history, int argc, char **argv,
+                                                    char **azColName) {
+        std::string id = argv[0];
+        query_handler::id.push_back(id);
```

Comment:
I kan constructe en string direkte i kaldet til push_back for at undgå at lave unødvendige temp variable, som alligevel bliver copied.

Reply:
done